src/main/kotlin/uk/gov/dluhc/printapi/service/PrintResponseFileService.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.dluhc.printapi.service
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import uk.gov.dluhc.printapi.printprovider.models.PrintResponses
+
+private val logger = KotlinLogging.logger {}
+
+@Service
+class PrintResponseFileService(
+    val sftpService: SftpService,
+) {
+    fun fetchAndUnmarshallPrintResponses(filePathToProcess: String): PrintResponses =
```

Comment:
Is this function (and also `removeRemoteFile`) adding anything by being in this bean? Could/should the `ProcessPrintResponseFileMessageListener` inject `SftpService` and call the methods directly ?

Reply:
Is the intention to abstract away the underlying implementation (in this case sftp) from the Listener?